### ISSUE TITLE
[v2][iOS] bottomTabs options not working. Stopped working on version 2.0.2476

### ISSUE THREAD

**Issue by rendomnet** [wix/react-native-navigation#4161]
### Issue Description

I have checked all version and 2.0.2475 is version where `bottomTabs visible: false` was working last time.

2.0.2585 did **NOT** fixed it. 
https://github.com/wix/react-native-navigation/issues/4112#issuecomment-430527100

### Steps to Reproduce / Code Snippets / Screenshots

```
Navigation.setDefaultOptions({
    bottomTabs: {
      visible: false,
      drawBehind: true,
      animate: false
    }
})
```

**update:**
![image](https://user-images.githubusercontent.com/18900210/47141374-5547a280-d2e2-11e8-85bc-afea738ac2d1.png)

Here is my bottomTabs navigator code:

`export const startMainNavigator = () => {
  Navigation.setRoot({
    root: {
      bottomTabs: {
        children: [
          {
            stack: {
              children: [
                {
                  component: {
                    name: "dh.Feed",
                    passProps: {
                      tabName: "Feed"
                    }
                  }
                }
              ],
              options: {
                bottomTab: {
                  text: "Tab 1",
                  icon: require("../assets/img/emptyTabIcon.png"),
                  testID: "Feed"
                },
                statusBar: {
                  visible: true,
                  animate: false,
                  drawBehind: true
                }
              }
            }
          },
          {
            stack: {
              children: [
                {
                  component: {
                    name: "dh.Inbox",
                    passProps: {
                      tabName: "Inbox"
                    }
                  }
                }
              ],
              options: {
                bottomTab: {
                  text: "Tab 2",
                  icon: require("../assets/img/emptyTabIcon.png"),
                  testID: "Inbox"
                },
                statusBar: {
                  visible: true,
                  animate: false,
                  drawBehind: true
                }
              }
            }
          },
          {
            stack: {
              children: [
                {
                  component: {
                    name: "dh.Log",
                    passProps: {
                      tabName: "Log"
                    }
                  }
                }
              ],
              options: {
                bottomTab: {
                  text: "Tab 3",
                  icon: require("../assets/img/emptyTabIcon.png"),
                  testID: "Log"
                },
                statusBar: {
                  visible: true,
                  animate: false,
                  drawBehind: true
                }
              }
            }
          },
          {
            stack: {
              children: [
                {
                  component: {
                    name: "dh.MyProfile",
                    passProps: {
                      tabName: "MyProfile"
                    }
                  }
                }
              ],
              options: {
                bottomTab: {
                  text: "Tab 4",
                  icon: require("../assets/img/emptyTabIcon.png"),
                  testID: "MyProfile"
                },
                statusBar: {
                  visible: true,
                  animate: false,
                  drawBehind: true
                }
              }
            }
          }
        ],
        options: {
          bottomTabs: {
            barStyle: "black",
            translucent: true,
            visible: false,
            drawBehind: false,
            animate: false,
            testID: "bottomTabsTestID",
            backgroundColor: "red"
          }
        }
      }
    }
  });
};`

---
### Environment
* React Native Navigation version: 2.0.2596
* React Native version: 0.56
* Platform(s) (iOS, Android, or both?): iOS
* Device info (Simulator/Device? OS version? Debug/Release?): both

**Comment by yogevbd** [comment]
Fixed in 2.0.2597

**Comment by LouiseReid** [comment]
I'm having this issue on v2.5.0,  no matter what option I pass in bottomTabs options none will take.

```   
 Navigation.setRoot({
      root: {
        bottomTabs: {
          options: {
            backgroundColor: 'red'
          },
          children: [
            {
              component: {
                name: 'navigationApp.FirstTabScreen',
                passProps: {
                  text: 'Home',
                },
                options: {
                  bottomTab: {
                    icon: images[0],
                    text: 'Home',
                    iconColor: 'darkblue',
                    selectedIconColor: 'lightblue'
                  },
                }
              },
            },
            {
              component: {
                name: 'navigationApp.SecondTabScreen',
                passProps: {
                  text: 'Search',
                },
                options: {
                  bottomTab: {
                    icon: images[1],
                    text: 'Search',
                    iconColor: 'darkblue',
                    selectedIconColor: 'lightblue'
                  }
                }
              },
            },
            {
              component: {
                name: 'navigationApp.ThirdTabScreen',
                passProps: {
                  text: 'Share',
                },
                options: {
                  bottomTab: {
                    icon: images[2],
                    text: 'Share',
                    iconColor: 'darkblue',
                    selectedIconColor: 'lightblue'
                  },
                }
              }
            },
          ],
        },
      }
    })
```
<img width="335" alt="screen shot 2019-01-07 at 13 45 55" src="https://user-images.githubusercontent.com/28352715/50771494-b4360b80-1282-11e9-96e1-98bed1c17ecb.png">
